setup-edly-locally.sh

```diff
@@ -0,0 +1,12 @@
+# Script to run commands for setup edly devstack locally
+docker-compose $DOCKER_COMPOSE_FILES exec -T lms bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform &&  ./manage.py lms setup_edly_multisite_devstack_Juniper_locally'
```

Comment:
In all below services we are following a specific format `setup_repo_service` can we manage it here as well ?
`setup_edly_edxapp_service` something like that ?

Reply:
Done